examples/templates/azure-windows/README.md

```diff
@@ -31,5 +31,34 @@ This template provisions the following resources:
 
 This means, when the workspace restarts, any tools or files outside of the data directory are not persisted. To pre-bake tools into the workspace (e.g. `python3`), modify the VM image, or use a [startup script](https://registry.terraform.io/providers/coder/coder/latest/docs/resources/script).
 
-> **Note**
+> [!NOTE]
 > This template is designed to be a starting point! Edit the Terraform to extend the template to support your use case.
+
+### Persistent VM
+
+> [!IMPORTANT]  
+> This approach requires the [`az` CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli#install) to be present in the PATH of your Coder Provisioner.
+> You will have to do this installation manually as it is not included in our official images.
+
+It is possible to make the VM persistent (instead of ephemeral) by removing the `count` attribute in the `azurerm_windows_virtual_machine` resource block as well as adding the following snippet:
+
+```hcl
+# Stop the VM
+resource "null_resource" "stop_vm" {
+  count      = data.coder_workspace.me.transition == "stop" ? 1 : 0
```

Comment:
count = data.coder_workspave.me.start_couny

Reply:
@matifali I'm not sure though, the issue is that if we use it for stopping, then it will look something like ` count = data.coder_workspave.me.start_count == 1 ? 0 : 1`, we should have `data.coder_workspave.me.stop_count` i think

Reply:
`"stop" ? 1 : 0` is fine. Adding a `stop_count` is too niche.